tests/ScopeTest.php

```diff
@@ -177,6 +178,42 @@ public function testContitionToWords()
         $this->assertEquals('Country that has reference Users where any referenced record exists', $country->scope()->toWords());
     }
 
+    public function testContitionUnsupportedToWords()
+    {
+        $condition = new Condition('name', 'abc');
```

Comment:
we should validate at create time (the operator)

Reply:
As mentioned above operator is now validated at condition construct time.

The wrong here is we do not know which model this condition is intended for to convert it to words.

Reply:
Hmm, is it complicated to print to words without model? If I construct some scope tree and I have not attached it yet, I would expect toWords still to work - of course, without specific typecasting, but we should support at least DateTime (instanceof) and default string. As simple as possible.

Reply:
You should pass the model as parameter, then it works (without attaching condition to model)

Reply:
![image](https://user-images.githubusercontent.com/2228672/88457844-eb323900-ce89-11ea-8384-aff454bf36e7.png)

what is usecase of this, why not simply getField?

Reply:
Should be okay with getField only

Reply:
You can theoretically print out the condition toWords without the model but it might not be the best looking string

Reply:
lets remove the `hasField()` now (must is not to fallback to null) and see if this strict implementation is not too strict in the real use

Reply:
done